docs/src/markdown-pages/guides/monetizing-your-loop.md

```diff
@@ -0,0 +1,41 @@
+---
+slug: 'guides/monetizing-your-loop'
+title: 'Monetizing Your Loop'
+description: 'Learn how to monetize your Loop with Pi.'
```

Comment:
I made this description text up - we may want something snazzier from marketing / product.

Reply:
I would follow up with Robin or Michelle and see if they have any input